.circleci/config.yml

```diff
@@ -22,41 +22,46 @@ executors:
 commands:
   restore-cargo-cache:
     steps:
-      # Restore cargo cache before installing anything with the cargo command (ie cargo install ...)
       - restore_cache:
-          name: Restore cargo cache
-          keys:
-            - cargo-{{ checksum "Cargo.lock" }}
+          name: Restore cargo registry cache
+          key: cargo-{{ checksum "Cargo.lock" }}
+          # Don't use fallback key to prevent registry cache from growing indefinitely.
```

Comment:
What does this mean?

Reply:
Clarifying why we don't use fallback key there. It can theoretically cause a cache to grow more and more over time since there is no function to prune cargo's registry.